uint8 address
uint8 messageType
uint8 messageData
uint8 messageData2
bool expectDataBack
